group-generators/generators/gitcoin-grant-15-donors/index.ts

```diff
@@ -0,0 +1,117 @@
+import { BigNumberish } from "ethers";
+import { dataOperators } from "@group-generators/helpers/data-operators";
+import { dataProviders } from "@group-generators/helpers/providers";
+import BigQueryProvider, {
+  SupportedNetwork,
+} from "@group-generators/helpers/providers/big-query/big-query";
+import { ValueType, Tags, FetchedData, GroupWithData } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Weekly,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const bigQueryProviderEthereum = new BigQueryProvider({
+      network: SupportedNetwork.MAINNET,
+    });
+    const bigQueryProviderPolygon = new BigQueryProvider({
+      network: SupportedNetwork.POLYGON,
+    });
+
+    const restProviderZkSync = new dataProviders.RESTProvider();
```

Comment:
By calling "restProviderZkSync" i have the feeling the provider is somehow related to zksync. Whereas here it's just a rest provider